iotdb/src/main/java/org/apache/iotdb/db/engine/filenodeV2/FileNodeManagerV2.java

```diff
@@ -200,22 +215,30 @@ private void writeLog(TSRecord tsRecord, boolean isMonitor, WriteLogNode logNode
   /**
    * update data.
    */
-  public void update(String deviceId, String measurementId, long startTime, long endTime,
+  public boolean update(String deviceId, String measurementId, long startTime, long endTime,
       TSDataType type, String v) {
+    if (rejectWrite) {
+      return false;
+    }
     // TODO
+    return false;
   }
 
   /**
    * delete data.
    */
-  public void delete(String deviceId, String measurementId, long timestamp)
+  public boolean delete(String deviceId, String measurementId, long timestamp)
       throws FileNodeManagerException {
+    if (rejectWrite) {
+      return false;
```

Comment:
should we block `delete` if disk space is not enough ?

Reply:
> should we block `delete` if disk space is not enough ?

because delete is in the lazy mode... It does not delete data on disk immediately.